### PR TITLE
feat: Support Contextual Tuples in Assertions API

### PR DESCRIPTION
**Pull request by smeadows-okta**
Added support for context assertions API.
## Description

Assertions are pure (object, relation, subject) tuples. This feature add contextual tuples for advanced scenarios with contextual based access

## References
[946](https://github.com/openfga/openfga/issues/946)

## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [x] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: pkg/server/test/write_assertions.go
Comment (jon-whit) [inline pkg/server/test/write_assertions.go:1]: Please add a test that asserts an error is returned from the Server's `WriteAssertions` API if more than 20 contextual tuples are included in the request.

Comment (smeadows-okta) [reply pkg/server/test/write_assertions.go:1]: 👍

File: internal/server/commands/commands.go
Comment (miparnisari) [inline internal/server/commands/commands.go:1]: can we please do the moving/renaming in a different PR?

**Comment by miparnisari** [comment]
Superseded by https://github.com/openfga/openfga/pull/1821

**Pull request closed without merging**